indico/modules/users/templates/dashboard.html

```diff
@@ -49,6 +49,7 @@ <h3>{{ user.full_name }}</h3>
                                 </div>
                             {% endif %}
                             {{ labels(user) }}
+                            {% block last_login_dt %} {% endblock %}
```

Comment:
The name is too specific. Please find a generic name. `after_user_labels` maybe. Also, no need to put a space in the default content.